# HG changeset patch
# User Rik <rik@octave.org>
# Date 1362082043 28800
#      Thu Feb 28 12:07:23 2013 -0800
# Node ID 34898b3fc32abd266c8c77134d9eaa8733056996
# Parent  23af74103b2c40ae317615c03b3ffed2a195edc0
build: Fix unused variable warning in compiling input.cc.

* libinterp/interpfcn/input.cc(get_user_input): Remove unused len variable.

diff --git a/libinterp/interpfcn/input.cc b/libinterp/interpfcn/input.cc
--- a/libinterp/interpfcn/input.cc
+++ b/libinterp/interpfcn/input.cc
@@ -351,18 +351,16 @@ get_user_input (bool& eof)
   eof = false;
 
   std::string retval;
 
   if (get_input_from_eval_string)
     {
       retval = current_eval_string;
 
-      size_t len = retval.length ();
-
       // Clear the global eval string so that the next call will return
       // an empty character string with EOF = true.
       current_eval_string = "";
 
       eof = true;
     }
   else
     retval = octave_gets (eof);
